VogtObserver3/Test.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver3
+{
+    class Test
```

Comment:
Might want a more explicit name. Maybe `ObserverTest` would be better? To indicate that this class is essentially going to test the observer pattern.

Reply:
Always a balance of too generic, to too verbose. `Test` is too generic. `TestObserverForPracticeProgram` is too verbose.

`TestObserver` is a nice middle =p

Reply:
Here's another thing to consider.

Sure, this is the `Test` class in the `VogtObserver3` program, so there is some level of assumption that could be made.. BUT the problem is `Test` is a term used throughout all of programming/computer science. Calling something just `Test`, when your application should eventually have automated`Unit Tests` and `Integration Tests` can be misleading.